app/controllers/lawyers_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LawyersController < ApplicationController
+  def index
+    @lawyer = Lawyer.all
+  end
+
+  def create
+    @current_user = current_user
+    @lawyer = current_user.Lawyer.new(lawyer_params)
+    @lawyer.save
+    redirect_to lawyers_path
+  end
+
+  def update
+    @current_user = current_user
+    @lawyer = current_user.Lawyer.new.find(params[:id])
+    @lawyer.update(lawyer_params)
+    redirect_to lawyers_path
+  end
```

Comment:
The same here, we need to render to JSON, and show the result of the operation in JSON format
```suggestion
  def update
    @lawyer = current_user.Lawyer.new.find(params[:id])
    if @lawyer.update(lawyer_params)
      render json: { result: 'Lawyer updated successfully' }
    else
      render json: { result: 'Something went wrong' }, status: :unprocessable_entity
    end
  end
```